sdk/servicebus/Microsoft.Azure.WebJobs.Extensions.ServiceBus/tests/WebJobsServiceBusTestBase.cs

```diff
@@ -215,5 +217,12 @@ internal async Task WriteTopicMessage(string message, string sessionId = null)
             }
             await sender.SendMessageAsync(messageObj);
         }
+
+        internal static void AssertNoLoggedErrors(IHost host)
```

Comment:
Wonder if we can do this automatically on StopAsync.

Can we add an https://docs.microsoft.com/en-us/dotnet/api/microsoft.extensions.hosting.ihostedservice?view=dotnet-plat-ext-5.0 implementation that does this as part of the shutdown?